cmd/nginx-ingress/main.go

```diff
@@ -440,9 +443,11 @@ func main() {
 		bytes := configs.GenerateCertAndKeyFileContent(secret)
 		nginxManager.CreateSecret(configs.DefaultServerSecretName, bytes, nginx.TLSSecretFileMode)
 	} else {
-		_, err = os.Stat("/etc/nginx/secrets/default")
+		_, err = os.Stat(configs.DefaultServerSecretPath)
```

Comment:
Can we write a unit test for this logic?

Reply:
good idea 👍  there was a bug